Automattic/jetpack
# Simple Payments: Fix misspelled dir

From #20629

#### Changes proposed in this Pull Request:

* Fixed misspelled dir

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* None really. Can installed an older version of Jetpack, add a simple payment block, then upgrade and ensure it still does.
*
